gatsby-node.ts

```diff
@@ -138,55 +157,67 @@ exports.createPages = async ({ actions, graphql }) => {
   const { createPage } = actions
 
   const pages = await getFolderEdges('index', graphql)
-  pages.forEach((edge) => createPageFromEdge(edge, createPage))
+  if (pages) {
+    pages.forEach((edge) => createPageFromEdge(edge, createPage))
+  }
```

Comment:
Can we change it to this on all places instead of adding if statements?

```suggestion
 pages?.forEach((edge) => createPageFromEdge(edge, createPage))
```